alpha/nodejs-chart/templates/buildconfig.yaml

```diff
@@ -0,0 +1,58 @@
+{{- if .Values.build.enabled }}
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata:
+  name: {{ include "nodejs.name" . }}
+  labels:
+    {{- include "nodejs.labels" . | nindent 4 }}
+spec:
+  output:
+    to:
+      kind: ImageStreamTag
+      name: {{ include "nodejs.imageName" . }}
+  source:
+    type: Git
+    git:
+      uri: {{ required "value 'build.uri' is required" .Values.build.uri }}
+      ref: {{ .Values.build.ref }}
+{{- if .Values.build.contextDir }}
+    contextDir: {{ .Values.build.contextDir }}
+{{- end }}
+{{- if and .Values.build.native.useDefaultDockerfile (eq .Values.build.mode "native") }}
+    dockerfile: |-
+      FROM registry.redhat.io/ubi8/nodejs-12:latest
+      USER root
+      RUN groupadd -r node && useradd -r -g node node
+      WORKDIR "/app"
+      COPY package*.json ./
+      RUN npm install
+      COPY . /app
+      ENV PORT 8080
+      EXPOSE 8080
+      USER node
+      CMD ["npm", "start"]
+{{- end }}
```

Comment:
This Dockerfile probably isn't necessary for NodeJS, as we have a supported s2i image that essentially does this same thing. Recommend just doing a single git input with an s2i strategy build